packages/react-components/src/templates/ResearchOutputTeamForm.tsx

```diff
@@ -321,6 +321,7 @@ const ResearchOutputPage: React.FC<ResearchOutputPageProps> = ({
                 onChangeTeams={setTeams}
                 getTeamSuggestions={getTeamSuggestions}
                 isEditMode={isEditMode}
+                authorsRequired={publishingEntity === 'Working Group'}
```

Comment:
Why is there a condition here? Won't the Team Output form only deal with Team Outputs?

Reply:
We can either temporarily send a prop from the WorkingGroupOutputs component and have that prop added in the TeamForm, which will have to be removed once we add the WorkingGroupForm, or have this in place, which will also have to be removed once we add the WorkingGroupForm.